### PR TITLE
Improve django action devex

### PR DESCRIPTION
**Pull request by AaDalal**
Rationale: 
1. it would be nice to see if tests pass before we go back and fix linting :)
2. it would be nice to see why linting failed (since github actions env may differ from local results)

### CODE REVIEW

File: django-check/action.yaml [hunk 1]
```diff
@@ -50,13 +56,7 @@ runs:
...
         if: ${{ inputs.black }}
-      - name: Test (run in parallel)
-        shell: bash
-        run: |-
-          cd ${{ inputs.path }}
-          pipenv run coverage run --concurrency=multiprocessing manage.py test --settings=${{ inputs.projectName }}.settings.ci --parallel
-          pipenv run coverage combine
-  # container:
-  #     image: python:${{ inputs.pythonVersion }}
+  container:
```
Comment (joyliu-q) [inline django-check/action.yaml:61]: Any reason we want to bring this back?

File: django-check/action.yaml [hunk 2]
```diff
@@ -50,13 +56,7 @@ runs:
         shell: bash
         run: |-
           cd ${{ inputs.path }}
-          pipenv run black --check .
+          pipenv run black --check . || pipenv run black --diff .
```
Comment (joyliu-q) [inline django-check/action.yaml:59]: This looks great! Thanks for adding this

File: django-check/action.yaml [hunk 3]
```diff
@@ -40,6 +40,12 @@ runs:
           cd ${{ inputs.path }}
           pip install pipenv
           pipenv install --deploy --dev
+      - name: Test (run in parallel)
+        shell: bash
+        run: |-
+          cd ${{ inputs.path }}
+          pipenv run coverage run --concurrency=multiprocessing manage.py test --settings=${{ inputs.projectName }}.settings.ci --parallel
```
Comment (joyliu-q) [inline django-check/action.yaml:47]: What's the rationale for moving this first? Usually, it's good practice to run the linter locally on your computer before making a PR change, so we can eliminate the "see if tests fail or not" by just making sure the code is linted beforehand.

The rationale is that if your code fails lint, you should lint it and push before we want to run the tests.

If you want to see, on the event that the test has failed lint, also see if the tests also pass or fail, you can use the `continue-on-error` flag with the linter and make sure you ultimately fail this step.

**Review by joyliu-q — CHANGES_REQUESTED** [review]
Have a few comments. Once this is merged, you will need to update your local action tag to reflect this new change.

File: django-check/action.yaml (hunk 1 above)
Comment (AaDalal) [reply django-check/action.yaml:61]: Do all products use the same python version? If I remember correctly, that's not true, which is the rationale for bringing this back.

File: django-check/action.yaml (hunk 3 above)
Comment (AaDalal) [reply django-check/action.yaml:47]: The use case is that (at least in labs) we often lint as a last step, but we need to test incrementally.

It seems like `continue-on-error` is a better approach here. I'll add that and re-request.

File: django-check/action.yaml (hunk 1 above)
Comment (AaDalal) [reply django-check/action.yaml:61]: I'm curious why we removed it, though

Comment (joyliu-q) [reply django-check/action.yaml:62]: Let's add it back temporarily and monitor closely.

**Review by joyliu-q — APPROVED** [review]
LGTM, let's monitor the container image temporarily

**Pull request merged**